src/p5.dance.js

```diff
@@ -636,6 +636,36 @@ module.exports = class DanceParty {
     const maxY = 400 - 40;
     const radiansToDegrees = 180 / Math.PI;
     const maxCircleRadius = 165;
+    function createColumn(group, location) {
+      for (let i=0; i<count; i++) {
```

Comment:
where is `count` coming from? Can you pass it in as an argument to make the scope clearer?